routetable/route_table.go

```diff
@@ -122,6 +122,11 @@ func (r *RouteTable) SetRoutes(ifaceName string, targets []Target) {
 	r.dirtyIfaces.Add(ifaceName)
 }
 
+func (r *RouteTable) QueueResync() {
+	r.logCxt.Info("Queueing a resync.")
+	r.inSync = false
```

Comment:
(+ agree that netlink monitoring would be a big increase in complexity here, so I'm inclined against it unless we really need it)